### PR TITLE
changed status APIs

### PR DESCRIPTION
**Pull request by madhuryaalur**

### CODE REVIEW

File: StatusAPIs.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    # Get it from Sentry
+    sentry_id = ''
+    camera = 'Cam1'
+    API_KEY = ""
+    URL = "https://api.smartsentry.ai/v2/status"
+
+    api_object = StatusAPIs(sentry_id, camera, API_KEY, URL)
+
+    result = api_object.get_results_for_status_api()
+    api_object.parse_output(result)
```
Comment (keshavamurali) [inline StatusAPIs.py]: Please change it to parse_get_output

File: StatusAPIs.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    # post response
+    res = {"person": True, "face": True}
+    camera_id = sentry_id+"_"+camera
+    api_object.update_payload_with_more_info(camera_id, res)
```
Comment (keshavamurali) [inline StatusAPIs.py]: Pl change the name to update_payload

File: StatusAPIs.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    def get_results_for_status_api(self):
+        self.url += "?sentry_id=" + self.sentry_id + "&camera=" + camera
+        response = requests.request("GET", self.url, headers=self.headers, timeout=25)
+        return response
+
+    def update_payload_with_more_info(self, camera_id, camera_response):
+        cam_info = {camera_id: camera_response}
+        self.payload.update({"camera_information": cam_info})
+
+    def post_response_for_status_api(self):
```
Comment (keshavamurali) [inline StatusAPIs.py:27]: This is just post_status . Right ? What do you mean by post_response ?

File: StatusAPIs.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        response = requests.request("POST", self.url, data=json.dumps(self.payload), headers=self.headers, timeout=25)
+        return response
+
+    def parse_output_post(self, response):
+        output = response.text
+        print("The result after DB update is = ", output)
```
Comment (keshavamurali) [inline StatusAPIs.py]: Don't we need to parse output here ? What is the output

File: StatusAPIs.py (hunk 3 above)
Comment (madhuryaalur) [reply StatusAPIs.py:27]: I meant post response from Blue Iris. It can be post status

File: StatusAPIs.py (hunk 4 above)
Comment (madhuryaalur) [reply StatusAPIs.py]: The lambda is currently just returning "updated" keyword as response

File: StatusAPIs.py (hunk 1 above)
Comment (madhuryaalur) [reply StatusAPIs.py]: done

File: StatusAPIs.py (hunk 2 above)
Comment (madhuryaalur) [reply StatusAPIs.py]: done

**Pull request merged**